config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
```

Comment:
This defines the routes we want to block correct? So I'd think this would be a `BlacklistConstraint`. Also, could you wrap this in the Katello module for namespacing and include something about routing to prevent confusing this with say filtering that uses the idea of blacklist/whitelist? e.g.

```
module Katello
  module RouteBlacklistConstraint
```